boring/src/ssl/callbacks.rs

```diff
@@ -521,3 +521,25 @@ where
         Err(err) => err.0,
     }
 }
+
+pub(super) unsafe extern "C" fn raw_info_callback<F>(
+    ssl: *const ffi::SSL,
+    mode: c_int,
+    value: c_int,
+) where
+    F: Fn(&SslRef, i32, i32) + Send + Sync + 'static,
```

Comment:
Shouldn't those `i32` be `Ssl3AlertLevel` and `SslInfoCallbackMode`

Reply:
The latter `i32` can be a variety of different things depending on the `mode`, I'll encode that in an enum with some helpers to make it more ergonomic for the user